Doc/library/zipfile.rst

```diff
@@ -818,7 +818,18 @@ Instances have the following methods and attributes:
 .. attribute:: ZipInfo.date_time
 
    The time and date of the last modification to the archive member.  This is a
-   tuple of six values:
+   tuple of six values representing the "last mod file time" and "last mod file date"
+   fields from the ZIP file's central directory.
+
+   .. note::
```

Comment:
I would move this note further down as "a tuple of six values" is lost as context when you get to "The tuple" below.

Reply:
The note also seems very similar to the one below. I would merge the information in this note into the one below.